options.go

```diff
@@ -47,9 +47,14 @@ type RequestOptions struct {
 
 // TransportOptions are transport related options.
 type TransportOptions struct {
-	ServiceName  string   `short:"s" long:"service" description:"The TChannel/Hyperbahn service name"`
-	HostPorts    []string `short:"p" long:"peer" description:"The host:port of the service to call"`
-	HostPortFile string   `short:"P" long:"peer-list" description:"Path of a JSON file containing a list of host:ports"`
+	ServiceName      string            `short:"s" long:"service" description:"The TChannel/Hyperbahn service name"`
+	HostPorts        []string          `short:"p" long:"peer" description:"The host:port of the service to call"`
+	HostPortFile     string            `short:"P" long:"peer-list" description:"Path of a JSON file containing a list of host:ports"`
+	CallerOverride   string            `long:"caller" description:"Caller will override the default caller name (which is yab-$USER)."`
```

Comment:
Mention that this is ignored if benchmarking? Or mention in benchmark options somewhere that "using this disables --caller".